File: 1010.py
a = input().split()
b = input().split()

x1 = int(a[1]) * float(a[2])
x2 = int(b[1]) * float(b[2])

valor = x1 + x2
print('VALOR A PAGAR: R$ {:.2f}'.format(valor))

File: 1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var a = lines.shift().split(' ');
var a1 = parseInt(a[1]);
var a2 = parseFloat(a[2]);

var b = lines.shift().split(' ');
var b1 = parseInt(b[1]);
var b2 = parseFloat(b[2]);

var valor = (a1 * a2) + (b1 * b2);
console.log('VALOR A PAGAR: R$ ' + valor.toFixed(2));

